SequenceOfNumbers.java

```diff
@@ -1,27 +1,46 @@
 import java.util.Scanner;
 
 public class SequenceOfNumbers {
-  public static void main(String[] args) {
-    System.out.println("Enter your sequence (use space between numbers):");
-    Scanner in = new Scanner(System.in);
-    String sequence = in.nextLine();
-    String numStr[] = sequence.split(" ");
+  /** 
+   * The function is used for determination type of sequence: non-decreasing or not non-decreasing
+   *
+   * @param newArgs - array of strings which contain testing sequence. One elemet of array - one number
+   */ 
+  public String сheckSequence(String[] newArgs) {

```

Comment:
single responsibility violation - method must check, not check & output & verify